samples/apps/copilot-chat-app/webapi/Program.cs

```diff
@@ -42,11 +46,16 @@ public static async Task Main(string[] args)
 
         // Configure middleware and endpoints
         WebApplication app = builder.Build();
+        app.UseDefaultFiles();
```

Comment:
Do we have a specific reason to enabled static files? If not, let's remove this to reduce the security surface we need to worry about.

Reply:
nothing specific, this is from their starter samples. Shall remove.